src/components/TabQueryEditor.vue

```diff
@@ -360,10 +368,14 @@
           const query = this.deparameterizedQuery
           this.$modal.hide('parameters-modal')
 
-          const runningQuery = this.connection.query(query);
+          const runningQuery = this.connection.query(query)
+          const queryStartTime = +new Date()
           const results = await runningQuery.execute()
+          const queryEndTime = +new Date()
+          const queryExecutionTime = queryEndTime - queryStartTime
           let totalRows = 0
           results.forEach(result => {
+            result.executeTime = queryExecutionTime
```

Comment:
Seeing as we're not measuring this in the back-end. I'd not put this on `result`, but instead create a new top level entry in `data` - `executeTime`. It's also not per-query, so this makes more sense to me.

Then in the computed it would just be `executeTime` rather than `result.executeTime`

Reply:
updated now